requirements.txt

```diff
@@ -1,10 +1,13 @@
+cx_Oracle >= 7.1.0
 numpy>1.7,<1.12; python_version == '2.6'
 numpy>1.7,<1.12; python_version == '3.2'
 numpy>1.7,<1.12; python_version == '3.3'
 numpy>1.7,<1.15; python_version == '2.7'
 numpy>1.7; python_version >= '3.4'
+pandas >= 0.24.1
```

Comment:
is this specific version needed?
`cc7` available is `0.17.1-1.el7`

Reply:
It’s definitely needs to be higher than that, I’ll try to see if I can identify the lowest version possible

Reply:
alternatively, is `pandas` needed, or can native tools suffice?

Reply:
Yes `pandas` is needed as everything so far has been built around the `dataframe` class.

Reply:
I was not expecting the OS version to work based on the function documentation from what is provided on the `pandas` API...but surprisingly it does work.  So I should remove the requirement or specify `>=` version available in `cc7`?

Reply:
I would not specify a version, just the package name (`RPM` package is `python-pandas`, not `pandas`), also for `sqlalchemy`)